Source/Core/FeatureDetection.js

```diff
@@ -318,6 +318,16 @@ var FeatureDetection = {
   typedArrayTypes: typedArrayTypes,
 };
 
+/**
+ * Detects whether the current browser supports Basis Universal textures and the web assembly modules needed to transcode them.
+ *
+ * @param {Scene} scene
+ * @returns {Boolean} true if the browser supports web assembly modules and the scene supports Basis Universal textures, false if not.
```

Comment:
Note - if we support the RGBA fallback we won't need `scene.context.supportsBasis` anymore.